src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -1007,20 +996,22 @@ public ReplayPosition call()
 
         private Flush(boolean truncate)
         {
+            if (logger.isTraceEnabled())
+                logger.trace("Creating flush task {}@{}", hashCode(), name);
             // if true, we won't flush, we'll just wait for any outstanding writes, switch the memtable, and discard
             this.truncate = truncate;
 
             metric.pendingFlushes.inc();
-            /**
-             * To ensure correctness of switch without blocking writes, run() needs to wait for all write operations
-             * started prior to the switch to complete. We do this by creating a Barrier on the writeOrdering
-             * that all write operations register themselves with, and assigning this barrier to the memtables,
-             * after which we *.issue()* the barrier. This barrier is used to direct write operations started prior
-             * to the barrier.issue() into the memtable we have switched out, and any started after to its replacement.
-             * In doing so it also tells the write operations to update the commitLogUpperBound of the memtable, so
-             * that we know the CL position we are dirty to, which can be marked clean when we complete.
+            /*
+              To ensure correctness of switch without blocking writes, run() needs to wait for all write operations
+              started prior to the switch to complete. We do this by creating a Barrier on the writeOrdering
+              that all write operations register themselves with, and assigning this barrier to the memtables,
+              after which we *.issue()* the barrier. This barrier is used to direct write operations started prior
+              to the barrier.issue() into the memtable we have switched out, and any started after to its replacement.
+              In doing so it also tells the write operations to update the commitLogUpperBound of the memtable, so
+              that we know the CL position we are dirty to, which can be marked clean when we complete.
```

Comment:
nit: Perhaps it would be best to just eliminate the extra asterisk that makes this JavaDoc formatted. (We seems to keep the asterisk prefixes in some other places in this file, and I suppose there's always https://google.github.io/styleguide/javaguide.html#s4.8.6-comments)